coderd/rbac/authz.go

```diff
@@ -357,12 +379,38 @@ func NewAuthorizer(registry prometheus.Registerer) *RegoAuthorizer {
 		Buckets:   buckets,
 	})
 
-	return &RegoAuthorizer{
-		query:        query,
-		partialQuery: partialQuery,
+	scopeDecisionCounter := factory.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "coderd",
+		Subsystem: "authz",
+		Name:      "scope_enforcement_total",
+		Help:      "Scope evaluation outcomes keyed by decision, scope, and resource.",
+	}, []string{"decision", "scope", "resource", "outcome"})
```

Comment:
Do you have an estimate for the cardinality?

decision: 6
resource: max 41 (`$ grep Type: coderd/rbac/object_gen.go | wc -l`)
scope: seems unbounded?
outcome: 2